views/block.tmpl

```diff
@@ -205,7 +205,7 @@
 				<tr>
 					<th>Transactions ID</th>
 					<th class="text-right">Vote Version</th>
-					<th class="text-right">Last Block</th>
+					<th class="text-left">Last Block</th>
```

Comment:
Left align of this column doesn't look right.

Before:

![image](https://user-images.githubusercontent.com/9373513/78580798-90040a00-77f8-11ea-8446-415c461393c5.png)

This PR:

![image](https://user-images.githubusercontent.com/9373513/78580826-9e522600-77f8-11ea-998c-1ac5a335ba76.png)